content/en/docs/concepts/windows/user-guide.md

```diff
@@ -158,14 +158,13 @@ schedule Linux and Windows workloads to their respective OS-specific nodes.
 The recommended approach is outlined below,
 with one of its main goals being that this approach should not break compatibility for existing Linux workloads.
 
-If the `IdentifyPodOS` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is
-enabled, you can (and should) set `.spec.os.name` for a Pod to indicate the operating system
+Starting from 1.25, please set `.spec.os.name` for a Pod to indicate the operating system
```

Comment:
```suggestion
Starting from 1.25, you can (and should) set `.spec.os.name` for each Pod, to indicate the operating system
```

Reply:
do we specify what the behavior is if a user doesn't?

Reply:
Should we specify it in the doc? If we specify, it may encourage users from not setting it, which defeats the purpose.

Reply:
Ideally, put that detail in as a comment in the upstream comments for the Pod API (if that doesn't land until v1.26, it's OK).

Reply:
Fair enough, I'll add that info to the pod API docs once the master branch on k/k opens.